theme/templates/base.html

```diff
@@ -62,39 +62,52 @@
     {% nevercache %}
     {% if messages %}
     {% for message in messages %}
-    <div class="page-tip animated slideInDown">
-        <div class="container">
-            <div class="row">
-                <div class="col-md-12">
-                    <button type="button" class="close btn-close-message">&times;</button>  {# TODO: TESTING #}
-                    {% if 'verification email' in message.message %}
-                        <!-- Google Code for HydroShare Conversions Conversion Page -->
-                        <script type="text/javascript">
-                        /* <![CDATA[ */
-                        var google_conversion_id = 823458818;
-                        var google_conversion_label = "uwesCPCzjIABEIL404gD";
-                        var google_remarketing_only = false;
-                        /* ]]> */
-                        </script>
-                        <script type="text/javascript" src="//www.googleadservices.com/pagead/conversion.js">
-                        </script>
-                        <noscript>
-                        <div style="display:inline;">
-                        <img height="1" width="1" style="border-style:none;" alt="" src="//www.googleadservices.com/pagead/conversion/823458818/?label=uwesCPCzjIABEIL404gD&amp;guid=ON&amp;script=0"/>
+        {% if message.tags == "error" %}
```

Comment:
This is in base.html so it will affect any message with a type of error.  I looked through the codebase and all the messages with a type of error are indeed errors and should benefit from the styling update.